.github/workflows/get-version.yml

```diff
@@ -87,9 +87,17 @@ jobs:
           fi
 
           case "$BRANCHNAME" in
-            master | [2-9][0-9].[0-9][0-9].x | release* | hotfix*)
+            master | [2-9][0-9].[0-9][0-9].x)
               echo "release=1" >> $GITHUB_OUTPUT
               ;;
+            release* | hotfix*)
```

Comment:
```suggestion
            ^release-[2-9][0-9].[0-9][0-9]-next | ^hotfix-[2-9][0-9].[0-9][0-9]-next)
```

Reply:
for now, we ll stick to a larger release pattern since it gives us more options to test the pipelines.